README.rst

```diff
@@ -0,0 +1,301 @@
+Intel® Media SDK
+================
+
+.. contents::
+
+Intel® Media SDK provides a plain C API to access hardware-accelerated video decode, encode and filtering on Intel® Gen graphics hardware platforms. Implementation written in C++ 11 with parts in C-for-Media (CM).
+
+**Supported video encoders**: HEVC, AVC, MPEG-2, JPEG, VP9  
+
+**Supported video decoders**: HEVC, AVC, VP8, VP9, MPEG-2, VC1, JPEG, AV1  
+
+**Supported video pre-processing filters**: Color Conversion, Deinterlace, Denoise, Resize, Rotate, Composition  
+
+Media SDK is a part of Intel software stack for graphics:
+
+* `Linux Graphics Drivers <https://intel.com/linux-graphics-drivers>`_ - General Purpose GPU Drivers for Linux&ast; Operating Systems
+
+  * Visit `documentation <https://dgpu-docs.intel.com>`_ for instructions on installing, deploying, and updating Intel software to enable general purpose GPU (GPGPU) capabilities for Linux&ast;-based operating system distributions.
+
+Media SDK Support Matrix
+------------------------
+
+Pay attention that Intel® Media SDK lifetime comes to an end in a form it
+exists right now. In particular, 
+
+* API 1.35 is projected to be the last API of 1.x API series
+* Runtime library (libmfxhw64.so.1) is not planned to get support of new Gen platforms
+* Project is going to be supported in maintainence mode, critical fixes only
+
+All future development is planned to happen within
+[oneVPL](https://github.com/oneapi-src/oneVPL) library and its runtime
```

Comment:
fix link, that's .md syntax, need .rst syntax as follows:
```
`oneVPL <https://github.com/oneapi-src/oneVPL>`_
```

Reply:
Fixed